examples/TestInclusions/Podfile

```diff
@@ -0,0 +1,12 @@
+workspace 'Examples.xcworkspace'
+project 'TestInclusions.xcodeproj'
+platform :ios, '6.0'
+
+abstract_target 'TestInclusionsPods' do
+    pod 'SwrveConversationSDK', '~> 4.3.0'
+
+    target 'TestInclusions'
+    target 'TestInclusionsTests' do
+        pod 'OCHamcrest', '~> 5.0'
```

Comment:
Removed.  Sorry about that, I'm just so used to using OCHamcrest that I didn't even think about it!

(And here was me thinking I was showing off proper abstract_target usage too!)